immutablecollections/_immutableset.py

```diff
@@ -100,7 +100,8 @@ def immutableset(
             iteration_order.append(value)
         elif forbid_duplicate_elements:
             raise ValueError(
-                "Input collection has duplicate items and forbid_duplicate_elements=False"
+                "Input collection to `immutableset` contains "
+                "{!r} twice and forbid_duplicate_elements=True".format(value)
```

Comment:
Like the dict case, print all the duplicate elements, not just the first one you find.